src/App.ts

```diff
@@ -61,6 +62,9 @@ class App {
   };
 
   //todo: enable unknown route handling
+  public unknownRoute: IReturnsVoid = () => {
```

Comment:
- make it private,
- change `unknownRoute` -> `enableUnknownRouteHandling` for just naming consistency
- and most important initialise it in constructor() like I did for other methods otherwise it won't work